### ISSUE TITLE
Time Warp attack description

### ISSUE THREAD

**Issue by Lagrang3** [bitcoinops/bitcoinops.github.io#1859]
Hi.
In the topic about the Time Warp Attack it reads: 
```
... For the second epoch, miners can make it appear that the first block was mined two weeks
in the past and the last block was mined at present, giving an average of about 20 minutes per block ...
```
If the time stamp difference between the first and the last block is 2 weeks that gives an approximate
time of 10 minutes per block since `2016 = 2 weeks * 7 days * 144 daily blocks`.
If the attackers wants to cut in half the difficulty they should make it look like if 4 weeks have passed between
the first and the last block.

A note aside: I think by this reasoning the difficulty could be reduced at most by a factor of 4 on each difficulty period
because of the limit on the adjustment step defined in `pow.cpp` `CalculateNextWorkRequired`.

**Comment by murchandamus** [comment]
That should read "was mined four weeks in the past" instead of "was mined two weeks in the past"

**Comment by murchandamus** [comment]
I opened a pull request to fix it in #1860
